tools/config/go-templates/atlas_client.mustache

```diff
@@ -91,3 +92,22 @@ func UseUserAgent(userAgent string) ClientModifier {
 		return nil
 	}
 }
+
+// IsError checks if API returned known error type.
+func IsError(err error) (*Error, bool) {
```

Comment:
let's follow similar semantics to the std lib 
```suggestion
// AsError  if err matches Error it returns its value and true. Otherwise, it returns nil and false.
func AsError(err error) (*Error, bool) {
```

Reply:
Good point. Is is for booleans :(